pipeline_dp/pipeline_operations.py

```diff
@@ -157,6 +161,19 @@ def count_per_element(self, rdd, stage_name: str = None):
         return rdd.map(lambda x: (x, 1))\
             .reduceByKey(lambda x, y: (x + y))
 
+    def reduce_accumulators_per_key(self, rdd, stage_name: str = None):
+        """Reduce the input RDD so that all elements per each key are merged.
+
+        Args:
+          rdd: input rdd which contains tuples (key, accumulator)
+          stage_name: name of the stage
+
+        Returns:
+          An RDD of tuples (key, accumulator).
+
+        """
+        rdd.reduceByKey(lambda acc1, acc2: acc1.merge([acc2]))
```

Comment:
let's assume for now that merge takes a list (then let's discuss on accumulator framework how to allow acceptance of 1 accumulator)

Reply:
okay